cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -16,14 +16,14 @@
 
 function(find_and_configure_raft)
 
-    set(oneValueArgs VERSION FORK PINNED_TAG)
+    set(oneValueArgs VERSION FORK PINNED_TAG USE_FAISS_STATIC)
     cmake_parse_arguments(PKG "${options}" "${oneValueArgs}"
                           "${multiValueArgs}" ${ARGN} )
 
-    if(DEFINED CPM_raft_SOURCE OR NOT DISABLE_FORCE_CLONE_RAFT)
-      set(CPM_DL_ALL_CACHE ${CPM_DOWNLOAD_ALL})
-      set(CPM_DOWNLOAD_ALL ON)
-    endif()
+    # if(DEFINED CPM_raft_SOURCE OR NOT DISABLE_FORCE_CLONE_RAFT)
+    #   set(CPM_DL_ALL_CACHE ${CPM_DOWNLOAD_ALL})
+    #   set(CPM_DOWNLOAD_ALL ON)
+    # endif()
```

Comment:
@dantegd @robertmaynard and I were curious what this logic was for... it seems if `CPM_raft_SOURCE` is set, it forces `CPM_DOWNLOAD_ALL` to on, effectively ignoring `CPM_raft_SOURCE`?